org.freedesktop.Sdk.Extension.ocaml.yaml

```diff
@@ -0,0 +1,111 @@
+        url: https://github.com/containers/bubblewrap/archive/refs/tags/v0.8.0.tar.gz
+        sha256: 8ede2b605d5aaf68aaa6ef1a5264ba7a31108c98417a8f88d289d0b5fa820c1b
+    post-install:
+      - bwrap --version
+
+  - name: rsync
+    buildsystem: autotools
+    config-opts:
+      - "--disable-xxhash"
+    sources:
+      - type: archive
+        url: https://download.samba.org/pub/rsync/src/rsync-3.2.7.tar.gz
+        sha256: 4e7d9d3f6ed10878c58c5fb724a67dacf4b6aac7340b13e488fb2dc41346f2bb
+    post-install:
+      - rsync --version
+
+  - name: opam
+    buildsystem: simple
+    sources:
+      - type: file
+        dest-filename: opam
+        only-arches:
+          - x86_64
+        url: https://github.com/ocaml/opam/releases/download/2.1.5/opam-2.1.5-x86_64-linux
+        sha256: f3681b8c2416340724317dfaa2143563af7b325eae4750eb0ee6a279ef4c4b61
+      - type: file
+        dest-filename: opam
+        only-arches:
+          - aarch64 
+        url: https://github.com/ocaml/opam/releases/download/2.1.5/opam-2.1.5-arm64-linux
+        sha256: ddb5063104e46c2a6304b91afd71290e3cf589236e1ed0dbc4c9e60b84e776d3
+      - type: file
+        dest-filename: opam-2.2a
+        only-arches:
+          - x86_64
+        url: https://github.com/ocaml/opam/releases/download/2.2.0-alpha2/opam-2.2.0-alpha2-x86_64-linux
+        sha256: ab69dec6a66e1c0afac04252355f3ceac28d5005584f1ab87cc2734e269656f2
+      - type: file
+        dest-filename: opam-2.2a
+        only-arches:
+          - aarch64
+        url: https://github.com/ocaml/opam/releases/download/2.2.0-alpha2/opam-2.2.0-alpha2-arm64-linux
+        sha256: 6e76de709aa0d445cec95322c199bb03fb89a8386618e81080c78a10c0e290fa
+    build-commands:
+      - chmod +x opam
+      - chmod +x opam-2.2a
+      - mv opam* ${FLATPAK_DEST}/bin
+    post-install:
+      - opam --version
+      - opam-2.2a --version
+
+  - name: ocaml
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/ocaml/ocaml/archive/refs/tags/5.0.0.tar.gz
+        sha256: 72fa3d0ba19b82fcb9e6c62e0090b9d22e5905c4be0f94faf56904a9377a9e5b
+    post-install:
+      - ocaml --version
+
+  - name: dune
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/ocaml/dune/archive/refs/tags/3.12.1.tar.gz
+        sha256: 95f7f173bd0265a796600051c79d5f8cd51673a8896608411debd90ca4e15d75
+    build-commands:
+      - make release
+      - PREFIX=${FLATPAK_DEST} make install
+    post-install:
+      - dune --version
+
+  - name: scripts
+    sources:
+      - type: script
+        commands:
+          - export PATH=$PATH:/usr/lib/sdk/ocaml/bin
+        dest-filename: enable.sh
+    buildsystem: simple
+    build-commands:
+      - cp enable.sh ${FLATPAK_DEST}
+
+  - name: appdata
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/share/appdata
+      - cp org.freedesktop.Sdk.Extension.ocaml.appdata.xml ${FLATPAK_DEST}/share/appdata
```

Comment:
```suggestion
      - install -Dm644 org.freedesktop.Sdk.Extension.ocaml.appdata.xml ${FLATPAK_DEST}/share/appdata
```

Reply:
For some reason the suggested `install` command is causing an error on the next command:
```
Running: install -Dm644 org.freedesktop.Sdk.Extension.ocaml.appdata.xml ${FLATPAK_DEST}/share/appdata
Running: appstream-compose  --basename=org.freedesktop.Sdk.Extension.ocaml --prefix=${FLATPAK_DEST} --origin=flatpak org.freedesktop.Sdk.Extension.ocaml
Processing application org.freedesktop.Sdk.Extension.ocaml
Error loading AppData file: no file found for org.freedesktop.Sdk.Extension.ocaml
Error: module appdata: Child process exited with code 1
```

Reply:
edited. forgot the `-t`

Reply:
now it works properly, thanks!